mysql/const.go

```diff
@@ -488,6 +488,12 @@ var DefaultLengthOfMysqlTypes = map[byte]int{
 	TypeSet:    8,
 }
 
+// BitLengthPerByte is the bit number of a byte.
+const BitLengthPerByte = 8
+
+// RowIDBitLength is the bit number of a row id in TiDB.
+const RowIDBitLength = 64
```

Comment:
Why not define this in `TiDB`?

Reply:
You are right. These are not MySQL-specific. Which place should I put them to? Can I hardcode them?

Reply:
Put it in the pkg you used in TiDB?